### PR TITLE
BB-1192: Cherry pick customizations from ginkgo branch

### PR DESCRIPTION
**Pull request by giovannicimolin**
This PR cherry picks customizations from the old release branch into the newest hawthorn.1 branch.

**Reviewers:**
- [ ] @lgp171188

### CODE REVIEW

File: playbooks/analytics_sandbox.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+- name: Bootstrap instance(s)
```
Comment (lgp171188) [inline playbooks/analytics_sandbox.yml:1]: @giovannicimolin, this needs to be updated based on the hawthorn version of the analytics playbook. See https://github.com/open-craft/openedx-deployment/commit/cca980973364bf1136640e5649dd93f31de26296#diff-40c660c4b60b66b30b41740d0dd05ea3 and https://github.com/hks-epod/configuration/pull/6 for details.

**Comment by lgp171188** [comment]
@giovannicimolin, other than the single comment, everything looks good to me.

File: playbooks/analytics_sandbox.yml (hunk 1 above)
Comment (giovannicimolin) [reply playbooks/analytics_sandbox.yml:1]: @lgp171188 Can you check if I cherry-pick the correct changes?

**Comment by giovannicimolin** [comment]
@lgp171188 Cherry picked the changes related to that commit. Can you do a final check and leave a :+1: here?

**Review by lgp171188 — APPROVED** [review]
:+1: 
- [x] I did not test this as it can be tested only with a deployment
- [x] I read through the code
- [x] ~~I checked for accessibility issues~~ NA
- [x] ~~Includes documentation~~ NA

**Pull request merged**